### PR TITLE
doc: restyle doc

### PR DESCRIPTION
**Pull request by samedii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-# Rkale
+# rkale
 
 ## Install
-#### Install Rkale:  
+
+Install lantern from pypi using pip or poetry:
 ```bash
-pip install rkale
+poetry add rkale
```
Comment (JoarGruneau) [inline README.md:8]: Think the installation guide should be the simplest way to get it up and running.
And don't suggest the user to install things they don't need.

**Review by JoarGruneau — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (samedii) [reply README.md:8]: Well, we only support pyproject.toml now so pip is kind of not even supported?

Comment (JoarGruneau) [reply README.md:8]: That is for the project syncing, you are still free to install it how you want.
Also you can still do syncing without a pyproject config if you specify source and destination

Comment (samedii) [reply README.md:8]: Okay I thought you said you were removing it?

Comment (JoarGruneau) [reply README.md:8]: No was just a thought. Other people thought it useful to be able to sync with checks without a project config, so left it in.
So maybe leave the global example in the README too?

**Pull request merged**